build.gradle.kts

```diff
@@ -226,7 +226,7 @@ fun Test.sharedIntegrationTestConfig(buildCodyDir: File, mode: String) {
       "CODY_TELEMETRY_EXPORTER" to "testing",
       // Fastpass has custom bearer tokens that are difficult to record with Polly
       "CODY_DISABLE_FASTPATH" to "true",
-  )
+      "ide.browser.jcef.headless.enabled" to "true")
```

Comment:
it still complains... 🤔 

```
[   6779]   WARN - #c.i.u.j.JBCefApp - JCEF is manually disabled in headless env via 'ide.browser.jcef.headless.enabled=false'
```

Reply:
I don't think these are environment properties, but Registry ones?

`com. intellij. openapi. util. registry.Registry.get("ide.browser.jcef.headless.enabled").setValue("true")` something like that...

Reply:
Ping @mkondratek

Reply:
Thanks Dominic. I tried setting this in the test setup. Something goes wrong in the test execution in CI. I have no time to focus on it now. Hopefully, I will get back to it later this week.